hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpContractStoresSubcommand.java

```diff
@@ -167,28 +196,125 @@ public static ContractKeyLocal from(ContractKey ckey) {
      * virtual merkle tree and it does the traversal on multiple threads.  (So the visitor needs to be able to handle
      * multiple concurrent calls.)
      */
-    boolean iterateThroughContractStorage(BiConsumer<ContractKey, IterableContractValue> visitor) {
+    boolean iterateThroughContractStorage(BiConsumer<ContractKeyLocal, UInt256> visitor) {
+
+        System.err.printf("iterateThroughContractStorage: start%n");
```

Comment:
so, first, this is the CLI, and these aren't logging.  it just console spew.  and ... I did have a reason for `err` instead of `out`.  but darned if I can remember what it was.  Will replace.  In fact, at this point, will probably strip most of these out.

Reply:
> this is the CLI, and these aren't logging

I was using "error" and "info" statements too loosely. I expected to see `System.out` instead of `System.err`